IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRoad/Entities/IfcSectionedSurface/DocEntity.xml

```diff
@@ -4,15 +4,22 @@
 		<DocAttribute Name="Directrix" UniqueId="db8a44ee-100e-4d8f-a7d9-ea294311f7db" DefinedType="IfcCurve">
 			<Documentation>The curve used to define the sweeping operation</Documentation>
 		</DocAttribute>
-		<DocAttribute Name="CrossSectionPositions" UniqueId="6f036e83-ef2e-4f46-9b50-f3d049cf88af" DefinedType="IfcPointByDistanceExpression" AggregationType="1" AggregationLower="2" AggregationUpper="?">
-			<Documentation>List of distance expressions in sequentially increasing order paired with CrossSections, indicating the position of the corresponding section along the Directrix.</Documentation>
+		<DocAttribute Name="CrossSectionPositions" UniqueId="6f036e83-ef2e-4f46-9b50-f3d049cf88af" DefinedType="IfcAxis2PlacementLinear" AggregationType="1" AggregationLower="2" AggregationUpper="?">
+			<Documentation>Position coordinate systems in sequentially increasing order paired with _CrossSections_, indicating the position of the corresponding section along the _Directrix_.</Documentation>
```

Comment:
```suggestion
			<Documentation>Placement coordinate systems in sequentially increasing order paired with _CrossSections_, indicating the placement of the corresponding section along the _Directrix_.</Documentation>
```

I'd argue that *position* only defines a point, while *placement*  additionally defines coordinate axes.

Reply:
I was adopting documentation akin to IfcSectionedSpine. Position is used in the attribute name "CrossSectionPositions".  But I agree with your suggestion for documentation.

Reply:
Yeah, I thought about the attribute name as well; however decided to not change it - it still conveys sort of correct meaning. And I would try to refrain from changing attribute names a minute before midnight. Therefore, only documentation changes :)

Reply:
How about just "A list of placements" instead of "Position coordinate systems"? Position coordinate systems sounds kind of odd.

Reply:
If we are renaming the documentation, wouldn't it also make sense to rename the attribute? ie CrossSectionPlacements instead of CrossSectionPositions  And if we renamed the attribute, shouldn't we also rename the attribute in IfcSectionedSolidHorizontal ?  This would change xml serialization.  Position has been used for IfcSweptAreaSolid etc.  Placement instead of Position might retrospectively be a better choice, but there is also consistency.  I don't think the documentation should use a different term than the attribute name.